core/src/main/java/tech/pegasys/eth2signer/core/SignerTypes.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.eth2signer.core;
+
+public enum SignerTypes {
+  BLS,
+  ETH,
```

Comment:
why not SECP? this is consistent with how the other enum values are named. I guess most correct would be ETH_SECP since the secp signer is Ethereum specific.

Reply:
Done - but what is BLS? Eth_BLS?